typescript/nomad-deploy/src/incremental/index.ts

```diff
@@ -0,0 +1,60 @@
+import * as ethers from 'ethers';
+
+import { NomadContext } from '@nomad-xyz/sdk/src';
+
+function toCall(t: ethers.ethers.PopulatedTransaction) {
+  return { to: t.to!, data: ethers.utils.toUtf8Bytes(t.data!) };
+}
+
+export async function enrollSpoke(
```

Comment:
add return types to all functions

Reply:
needs JSDoc stating what it does exactly

Reply:
what is the network state intended to be at this point?

Reply:
could u pls check if that is fine at this point?